test/wizard.jl

```diff
@@ -116,8 +118,8 @@ end
     @test state.source_urls == ["http://127.0.0.1:14444/a/source.tar.gz"]
     @test state.source_hashes == [libfoo_tarball_hash]
     @test Set(state.compilers) == Set([:c, :rust, :go])
-    @test state.preferred_gcc_version == BinaryBuilder.available_gcc_builds[1]
-    @test state.preferred_llvm_version == BinaryBuilder.available_llvm_builds[4]
+    @test state.preferred_gcc_version == getversion(available_gcc_builds[1])
+    @test state.preferred_llvm_version == getversion(BinaryBuilder.available_llvm_builds[4])
```

Comment:
Why `[4]`?  Is that just `end`?  If so, we should use `[end]` instead.

Reply:
Yes, it should be `[end]`.  Initially in #553 I used `[4]` everywhere, then I realised it was just the last element and fixed, but forgot this one.  Thanks!

Reply:
Well, looking better at the code, it was better for this to be 4: it's matching the fact that the wizard emulator pressed arrow down three times.